src/OpenTracing/SpanOptions.php

```diff
@@ -43,7 +43,17 @@ public static function create(array $options)
 
```

Comment:
what is happening in line 41? `getChildOf` is typed as returning a `ChildOf` instance, but here it is set to an array